### DISCUSSION TITLE
How to uninstall Paru

### DISCUSSION THREAD

**Discussion by willyblaise** [Morganamilo/paru#385] [General]
I am wondering if anyone on here knows how to uninstall paru from Arch?  I tried to uninstall with:
paru -R paru
pacman -R paru
yay -R paru

None of the above works and i get the following error message so to speak.

//error
error: target not found: paru

I could reinstall Arch to take care of the issue but i would rather just uninstall and get back a 4th of my paltry hard drive space. 

thanks in advance

**Comment by Morganamilo** [comment]
You probably installed on of the other paru packages `paru-bin` `paru-git` or installed paru via cargo.
